### ISSUE TITLE
Connecting to a remotely tunnelled-to jlab?

### ISSUE THREAD

**Issue by timothydmorton** [dfm/Ganymede#5]
Would there be any way to use this app to directly run the jlab associated with a specified port?  The use case I'm thinking of is if I have opened an SSH tunnel to a remote jlab instance somewhere, and would like to run that free of the browser as well.  (Great idea with this app, btw, thanks!)

**Comment by dfm** [comment]
Yes - I think it would be awesome to be able to connect to an existing instance via a port number (or a URL perhaps), but that's not implemented yet. It shouldn't be too hard, but it hadn't been a priority so far!
